client/src/components/utils/Header/Menu/index.js

```diff
@@ -16,8 +16,8 @@ export default function Menu({ show = false }) {
     return (
       <Router>
         <div className="header--menu">
-          {links.map(({ target, label }) => (
-            <Link to={target} className="header--menu-text">
+          {links.map(({ target, label }, i) => (
+            <Link key={`label_${i}`} to={target} className="header--menu-text">
```

Comment:
@ahmedisam99 You can use text for keys too, so if it makes life easier you could use `label` as the key's value.